### PR TITLE
[5.2] Fixing docblock arguments

### PR DESCRIPTION
**Pull request by votemike**
Fixing some incorrect docblocks

### CODE REVIEW

File: src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php [hunk 1]
```diff
@@ -10,14 +10,14 @@ class AddQueuedCookiesToResponse
...
-     * @var \Illuminate\Cookie\CookieJar
+     * @var \Illuminate\Contracts\Cookie\QueueingFactory
      */
     protected $cookies;
 
     /**
      * Create a new CookieQueue instance.
      *
-     * @param  \Illuminate\Cookie\CookieJar  $cookies
+     * @param  \Illuminate\Contracts\Cookie\QueueingFactory  $cookies
```
Comment (GrahamCampbell) [inline src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php]: This doesn't match the actual typehint.

File: src/Illuminate/Encryption/McryptEncrypter.php [hunk 2]
```diff
@@ -29,7 +29,7 @@ class McryptEncrypter extends BaseEncrypter implements EncrypterContract
      * Create a new encrypter instance.
      *
      * @param  string  $key
-     * @param  int     $cipher
+     * @param  string  $cipher
```
Comment (GrahamCampbell) [inline src/Illuminate/Encryption/McryptEncrypter.php]: ?

File: src/Illuminate/Foundation/Console/OptimizeCommand.php [hunk 3]
```diff
@@ -118,8 +118,6 @@ protected function getTraverser()
      */
     protected function getClassFiles()
     {
-        $app = $this->laravel;
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/OptimizeCommand.php]: Please revert this.

File: src/Illuminate/Encryption/McryptEncrypter.php (hunk 2 above)
Comment (votemike) [inline src/Illuminate/Encryption/McryptEncrypter.php]: $cipher is a string, not an int.

Comment (GrahamCampbell) [inline src/Illuminate/Encryption/McryptEncrypter.php]: Oh yes, so it is.

**Comment by GrahamCampbell** [comment]
Please squash to one commit.

**Comment by crynobone** [comment]
Shouldn't this send to 5.1?

**Comment by GrahamCampbell** [comment]
That's true. I thought it was. Please target this to 5.1

**Comment by votemike** [comment]
Why 5.1 out of interest?
This is only my second time submitting a pull request to Laravel

**Comment by GrahamCampbell** [comment]
Because 5.2 isn't shipping until the end of November and 5.1 is an LTS release also.

**Comment by votemike** [comment]
So patches get merged up rather than down?

**Comment by GrahamCampbell** [comment]
Yes, always. It would be very hard to do the opposite.

**Pull request closed without merging**